pages/contribution-flow.js

```diff
@@ -132,6 +135,9 @@ class NewContributionFlowPage extends React.Component {
     }
 
     const contributionBLocker = getContributionBlocker(LoggedInUser, account, tier, Boolean(this.props.tierId));
+    if (contributionBLocker.reason === 'NO_CUSTOM_CONTRIBUTION') {
```

Comment:
It's better to import the constant and use it rather than hard-coding so that if it changes in the future we can just change it at a single location. 😄 

```suggestion
    if (contributionBLocker.reason === CONTRIBUTION_BLOCKER.NO_CUSTOM_CONTRIBUTION) {
```

Reply:
Oh.. 🧠 sure, any other changes should I make...